### PR TITLE
Add docs for GHC-87491

### LINKED ISSUE #39: PsErrImportPostQualified -> 87491

### PR DESCRIPTION
**Pull request by RaoulHC**
Fixes #39.

### CODE REVIEW

File: message-index/messages/GHC-87491/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Found 'qualified' in postpositive position
```
Comment (david-christiansen) [inline message-index/messages/GHC-87491/index.md:2]: While the user's guide uses the term "postpositive", I'm not sure that it's the world's most well-understood term. I hadn't heard it before today. What about "after the module"?

**Review by david-christiansen — COMMENTED** [review]
I'm  not so sure about the word "postpositive" - other than that, this is great!

File: message-index/messages/GHC-87491/index.md (hunk 1 above)
Comment (RaoulHC) [reply message-index/messages/GHC-87491/index.md]: Updated. Yeah I just followed the users guide, and wasn't sure if I'd somehow just managed to miss that terminology.

**Review by david-christiansen — APPROVED** [review]

**Comment by david-christiansen** [comment]
Thanks again!

**Pull request merged**